src/DataCollector/SeoCollector.php

```diff
@@ -76,16 +77,28 @@ public function lateCollect(): void
         $this->data['metaGooglebot'] = $this->robotDirectivesChecker->getMetaGooglebotsTag();
         $this->data['metaGooglebotNews'] = $this->robotDirectivesChecker->getMetaGooglebotNewsTag();
         $this->data['language'] = $this->robotDirectivesChecker->getLanguage();
+
+        $this->data = $this->cloneVar($this->data);
```

Comment:
Here, we clone all the data collected using the VarDumper.
This allow powerful serialization of objects to be printed on the profiler panel using the `profiler_dumper` function.
However, this implies some little things to know and how to tackles those, as you won't consume the collected data strictly the same way.